workers/Dockerfile

```diff
@@ -1,25 +1,14 @@
-FROM continuumio/miniconda3:4.3.11
-
-RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
-WORKDIR /home/user
-
-RUN apt-get install libcurl4-openssl-dev
-
-COPY workers/environment.yml .
-
-RUN conda env create -f environment.yml
-
-ENV PATH "/opt/conda/envs/workers/bin:$PATH"
+FROM ccdl/data_refinery_worker_base
 
 COPY data_models/dist/data-refinery-models-* data_models/
```

Comment:
The developer runs `python setup.py sdist` themselves when they've made changes to the data models. I included this in the workers' README.md: https://github.com/data-refinery/data_refinery/blob/master/data_models/README.md#developing